pori_python/ipr/main.py

```diff
@@ -294,7 +294,7 @@ def ipr_report(
     username: str,
     password: str,
     content: Dict,
-    ipr_url: str = DEFAULT_URL,
+    ipr_url: str,
```

Comment:
Hi @elewis2,
This is requiring me to enter a valid ipr_url and potentially get access errors, even though I am using 'ipr_upload' = False and have no interest in the ipr_conn functions or tests for the expression plot graphkb matches.

Couldn't the ipr_url be like the graphkb_url: str = "", below?